Makefile

```diff
@@ -227,14 +227,14 @@ install.tools: .install.gitvalidation .install.gometalinter .install.md2man .ins
 	fi
 
 .PHONY: \
+	bin/conmon \
```

Comment:
Aren't these not PHONY, since they are real files on disk?

Reply:
> Aren't these not PHONY, since they are real files on disk?

They're real files, but *this* `Makefile` doesn't know about their dependencies.  So keeping them in `.PHONY` here means we always execute `make` in the subdirectory, where the subdir `Makefile` is used to see if anything actually needs rebuilding.

The `.PHONY` names are unimportant, so these could stay `conmon` and `pause`.  I added the `bin/` prefix to match `bin/crio` (where this `Makefile does know the prereqs), but I can leave them as `conmon` and `pause` if you prefer.

Reply:
Thanks for the explanation.